components/Header/Header.js

```diff
@@ -10,3 +10,17 @@ export default function Header() {
     </>
   );
 }
+
+export const StyledHeader = styled.header`
+  width: 100vw;
+  text-align: center;
+  color: #629fb5;
+
+  h1 {
+    margin-bottom: 5px;
+  }
+
+  p {
+    margin: 10px, 10px, 10px, 10px;
```

Comment:
You could also write:

```suggestion
    margin: 10px;
```